src/category_theory/closed/monoidal.lean

```diff
@@ -261,6 +261,22 @@ def of_equiv (F : monoidal_functor C D) [is_equivalence F.to_functor] [h : monoi
       exact adjunction.left_adjoint_of_nat_iso i,
     end } }
 
+lemma of_equiv_curry_def (F : monoidal_functor C D) [is_equivalence F.to_functor]
```

Comment:
This just says "currying commutes with equivalence" but it's hard to state because you only know that D is monoidal closed and you are pulling this back to C via the equivalence. Why not add a docstring for this theorem and the next to make the statements easier to read for humans?